public/app/plugins/datasource/loki/LogContextProvider.test.ts

```diff
@@ -225,8 +230,9 @@ describe('LogContextProvider', () => {
         10,
         LogRowContextQueryDirection.Backward,
         {
-          expr: '{bar="baz"} | logfmt | line_format = "foo"',
```

Comment:
Realized this was incorrect in most line format expressions, so I fixed it. 

![line format](https://github.com/grafana/grafana/assets/1069378/86cf73e4-ba60-415d-8394-f477068c8890)

Also removed some type castings that were not really necessary. FYI @svennergr

Reply:
Nice, good 👀 !